client/components/Grid.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+// eslint-disable-next-line import/extensions
+import CardsList from './CardsList.jsx';
+// eslint-disable-next-line import/extensions
+import GridHeader from './GridHeader.jsx';
+
+const Container = styled.div`
+  background-color: rgb(250,250,250);
+  width: 100%;
+  max-width: 1440px;
+  max-height: 838px;
+  margin-top: 48px;
+  color: #101820;
+  font-family: Roboto, "Helvetica Neue", sans-serif;
+`;
+
+class Grid extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // eslint-disable-next-line no-undef
+      currentId: window.location.pathname.split('/')[1],
+      currentName: '',
+      currentCategory: '',
+      similarRestaurants: [],
+      photos: [],
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  componentDidMount() {
+    const { currentId } = this.state;
+    this.getInfo(currentId);
+  }
+
+  getInfo(id) {
+    axios.get(`/zagat/restaurants/${id}`)
+      .then((res) => {
+        this.setState({
+          currentName: res.data.restaurant.name,
+          currentCategory: res.data.restaurant.category,
+          similarRestaurants: res.data.similar,
+          photos: res.data.photos,
+        });
+      })
+      .catch((err) => {
+        throw err;
```

Comment:
Do you want to throw the err? Wouldn't that crash the whole app?

We might want to talk about this as a team, but it would be cool to render an error page here (although probably outside of the scope of our project :) )

Reply:
Good point, I was trying to remove the console.log - how did you handle your axios error?